src/library/scala/collection/ArrayOps.scala

```diff
@@ -239,6 +239,18 @@ final class ArrayOps[A](private val xs: Array[A]) extends AnyVal {
     */
   def lastOption: Option[A] = if(isEmpty) None else Some(last)
 
+  /** Compares the size of this array to a test value.
+    *
+    *   @param   otherSize the test value that gets compared with the size.
+    *   @return  A value `x` where
+    *   {{{
+    *        x <  0       if this.size <  otherSize
+    *        x == 0       if this.size == otherSize
+    *        x >  0       if this.size >  otherSize
+    *   }}}
+    */
+  def sizeCompare(otherSize: Int): Int = Integer.compare(xs.length, otherSize)
```

Comment:
I noticed that we also miss `sizeIs` (and `lengthIs`) in `ArrayOps`. That being said, this operation (as well as `sizeCompare` and `lengthCompare`) is not really useful on an `Array` because the size is always known. It could be useful for uniformity with other collections.

Reply:
because both `String` and `Array` have known lengths, `sizeIs`/`lengthIs` can just return their length, which will compare correctly